WebRoot/js/zimbraMail/abook/view/ZmContactPicker.js

```diff
@@ -420,12 +420,13 @@ function() {
             this._searchInSelect.addOption(ZmMsg.GAL, true, ZmContactsApp.SEARCHFOR_GAL);
         }
 
+		this._searchInSelect.addOption("Organizational Address Book", false, 4);
```

Comment:
this option should only be added in UI when HAB is enabled or root id is present for HAB

Reply:
Fixed.